src/gl.rs

```diff
@@ -1112,6 +1112,24 @@ pub fn get_program_iv(program: GLuint, pname: GLenum) -> GLint {
 }
 
 #[inline]
+pub fn get_vertex_attrib_iv(index: GLuint, pname: GLenum) -> GLint {
+    unsafe {
+        let mut result: GLint = 0 as GLint;
+        ffi::GetVertexAttribiv(index, pname, &mut result);
+        return result;
+    }
+}
+
+#[inline]
+pub fn get_vertex_attrib_fv(index: GLuint, pname: GLenum) -> Vec<GLfloat> {
+    unsafe {
+        let mut result = vec![0 as GLfloat; 4];
```

Comment:
In the [OpenGL 2.1 spec](https://www.opengl.org/sdk/docs/man2/xhtml/glGetVertexAttrib.xml) (the mode we open the OpenGL context for normal servo in desktop IIRC):

```
GL_INVALID_OPERATION is generated if index is 0 and pname is GL_CURRENT_VERTEX_ATTRIB.
```

So... This changes are correct, we need to emulate that somehow in the WebGL layer, not here.